src/sysext/sysext.c

```diff
@@ -148,6 +149,59 @@ static int parse_mutable_mode(const char *p) {
         return mutable_mode_from_string(p);
 }
 
+static int config_parse_mutable_mode(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        MutableMode *m = ASSERT_PTR(data);
+        int r;
+
+        assert(rvalue);
+
+        r = parse_mutable_mode(rvalue);
+        if (r < 0)
+                return log_syntax_parse_error(unit, filename, line, r, lvalue, rvalue);
+
+        *m = r;
+        return 0;
+}
+
+static int parse_config_file(ImageClass image_class) {
+        const char *section = image_class == IMAGE_SYSEXT ? "Sysext" : "Confext";
```

Comment:
Maybe better to not abbreviate the section name?
`SystemExtension` and `ConfigExtension` ?

Reply:
The "sysext" and "confext" naming is the most common form.  It seems better to follow that here too.

Reply:
Then, maybe SysExt and ConfExt?

Reply:
I would leave it as it is, honestly. But I will wait for your green light.

Reply:
It seems there's prior art for the camelcase version:
```console
$ git grep '[SC]ys[eE]xt'
NEWS:          StubPcrKernelImage and StubPcrInitRDSysExts EFI variables.
man/systemd-stub.xml:        <term><varname>StubPcrInitRDSysExts</varname></term>
src/boot/stub.c:                (void) efivar_set_uint64_str16(MAKE_GUID_PTR(LOADER), u"StubPcrInitRDSysExts", TPM2_PCR_SYSEXTS, 0);
```

Reply:
So do you want me to change it in SysExt and ConfExt?